src/lib/common/macroSubstitute.cpp

```diff
@@ -104,26 +103,29 @@ static void attributeValue(std::string* valueP, const std::vector<ContextAttribu
 *
 * macroSubstitute - 
```

Comment:
It is almost possible to review the new version based on the diff. I'd recommend to look to the file directly ("View" buttom above).

NTC (just informative)